README.md

```diff
@@ -71,6 +71,189 @@ const result = validate(value, validations)
 | null          |                   |                   |  
 | undefined     |                   |                   |  
 
+
+
+#### Contains
+
+`contains` -  The contains validator is useful for validating allowance or restriction in certain values.
+It checks that the given value exists in the target given by the **allowed** or  **notAllowed** option.
+
+You can specify the validator as a list, string or as an object (in which case the keys of the object are used).
+
+**allowed option examples:**
+
+```javascript
+var list = ["small", "medium", "large"]
+const value = 'small'
+var options = { allowed: list }
+const validations = { contains: options }
+const result = validate(value, validations) 
+/* {
+    value: 'small',
+    errors: []
+} */
+
+var list = ["small", "medium", "large"]
+const value = 'xlarge'
+var options = { allowed: list }
+const validations = { contains: options }
+const result = validate(value, validations) 
+/* {
+    value: 'xlarge',
+    errors: [{ notContains: ["small", "medium", "large"] }]
+} */
+
+var text = "hello world"
+const value = 'hello'
+var options = { allowed: text }
+const validations = { contains: options }
+const result = validate(value, validations) 
+/* {
+    value: 'hello',
+    errors: []
+} */
+
+var text = "lorem ipsum dolor"
+const value = 'hello'
+var options = { allowed: text }
+const validations = { contains: options }
+const result = validate(value, validations) 
+/* {
+    value: 'hello',
+    errors: [{ notContains: "lorem ipsum dolor" }]
+} */
+
+
+var object = { foo: true }
+const value = 'foo'
+var options = { allowed: object }
+const validations = { contains: options }
+const result = validate(value, validations) 
```

Comment:
this use case is not clear to me. when exactly someone would need a contains validator for object? 

is it for schema validator? if so, is it the right tool?

and why `contains` validates the keys and not the values?

Reply:
It is actually to check for the existence of a specific attribute on an object.
I changed the documentation and test samples to make it easier to understand

Reply:
i think we should move it to a different validator like `hasKeys` or something instead of bury this feature into `contains` validator
the idea would be to highlight this feature that could be important to validate complex objects

Reply:
I understand, but a functionality to validate keys ends up being a derivation of the validator itself to contain or not something.

Wouldn't it be better to leave it as is and in an evolution to validate a complex object to treat it separately? Even if this is in a possible duplicate validation?